src/main/docs/guide/annotations/conditional.adoc

```diff
@@ -0,0 +1,5 @@
+Since Micronaut Cache 4.2.0, the above annotations can be conditionally disabled via an https://docs.micronaut.io/latest/guide/#evaluatedExpressions[Expression Language] expression in the `condition` attribute.
+
+For example, we can cache the result of a method invocation only if the `id` parameter is greater than 5:
+
+snippet::io.micronaut.cache.ConditionalService[project="cache-caffeine", source="test", tags="conditional" indent=0]
```

Comment:
It does work

<img width="809" alt="image" src="https://github.com/micronaut-projects/micronaut-cache/assets/49317/9f356758-9dbf-4577-b47c-3d20f4296071">

But I will work on adding a `test-suite`, a `test-suite-groovy` and `test-suite-kotlin` as this module currently doesn't have them

Reply:
Ahhhh, I see...  it's because it goes in as a multi-language sample, so if the cookie doesn't say you want Java (ie: you have selected Kotlin in the past), then it hides it 😞

That's why the embedded source (not from a snippet) such as here

https://github.com/micronaut-projects/micronaut-cache/blob/2e6ba126e0a5a4cba0455c6ba505bdf88d2a626d/src/main/docs/guide/caffeine.adoc#L32-L45

Works...